src/plugins/dam/components/uploader/delegate.js

```diff
@@ -173,15 +173,21 @@ class Delegate extends AbstractDelegate {
   }
 
   handleCreditApplyAll(credit, files, initialValues) {
+    const config = { schemas, assetCount: Object.keys(files).length };
     const data = {
       fields: ['credit'],
       values: { credit },
     };
-    this.dispatch(initialize(this.getFormName(), { ...initialValues, credit }, 'credit'));
-    this.dispatch(patchAssets(data, files, this.config));
+    this.dispatch(initialize(
+      this.getFormName(),
+      { ...initialValues, credit: { value: credit } },
+      'credit',
+    ));
+    this.dispatch(patchAssets(data, files, config));
```

Comment:
@hunyoboy you just want to include the assetCount, right? seems like it would be better to do something like

`this.dispatch(patchAssets(data, files, { ...this.config, assetCount: Object.keys(files).length }));`

Reply:
@efjacobson nice works for me. do you really want to pass the whole config? i feel like schemas is enough currently. this is the same as the other flows ( ex. reorderGallery ). what do you think?

Reply:
@hunyoboy i suppose it doesnt really matter, i just know the uploader is kinda tricky and was wary about replacing the config